src/psyclone/tests/psyir/nodes/omp_directives_test.py

```diff
@@ -113,6 +114,72 @@ def test_ompparallel_changes_begin_string(fortran_reader):
     assert pdir.children[2] is not priv_clause
 
 
+def test_ompparallel_structures_private_clause():
+    '''TODO'''
+    subroutine = Routine("testsub")
+    parallel = OMPParallelDirective.create()
+    subroutine.addchild(parallel)
+    sing = OMPSingleDirective()
```

Comment:
pylint complains that `sing` is unused.

Reply:
Fixed by redoing the test to use fortran_reader